src/stepfunctions/steps/constants.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from enum import Enum
+from stepfunctions.steps.fields import Field
+
+# Path to SageMaker placeholder parameters
+placeholder_paths = {
+    # Paths taken from https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_CreateProcessingJob.html
```

Comment:
question: 

1. what does the comment mean when it says "taken" from the documentation link?
2. can all [properties](https://docs.aws.amazon.com/step-functions/latest/dg/connect-sagemaker.html) be represented by placeholders or is it only some?

Reply:
1: I took the location to the CreateProcessingJob request from there to save in placeholder_paths for each arg
2: all can